js/main.js

```diff
@@ -917,7 +925,7 @@ class App extends React.Component {
             title="Manage Environments"
             data-placement="bottom"
             className="btn btn-default"
-            disabled={!(this.state.connected && this.state.envID)}
+            disabled={!(this.state.connected && this.state.envID && !this.state.readonly)}
```

Comment:
This is great! Should also be applied to 986 to disable the env modal as well.

Reply:
Thank you for the review. Shall add the suggested change as well.